### PR TITLE
added card header style mixins

### PR DESCRIPTION
**Pull request by kossel**
Less mixin for card header style

### CODE REVIEW

File: components/style/themes/default.less [hunk 1]
```diff
@@ -264,3 +264,9 @@
 // ---
 @rate-star-color: #f5a623;
 @rate-star-bg: #e9e9e9;
+
+// Card
+// ---
+@card-head-height: 48px;
```
Comment (benjycui) [inline components/style/themes/default.less]: `head` => `header` ?

Comment (kossel) [inline components/style/themes/default.less]: Hmm the current class uses 'head' as name, just to make it consistent, should I change to header?

https://github.com/ant-design/ant-design/blob/feature-2.9/components/card/style/index.less#L22

File: components/card/style/index.less [hunk 2]
```diff
@@ -32,7 +33,7 @@
       width: 100%;
       overflow: hidden;
       white-space: nowrap;
-      color: @heading-color;
+      color: @card-head-bgcolor;
```
Comment (afc163) [inline components/card/style/index.less]: It should be `@card-head-color`?

File: components/style/themes/default.less [hunk 3]
```diff
@@ -264,3 +264,9 @@
...
+@card-head-color: @heading-color;
+@card-head-bgcolor: @component-background;
```
Comment (afc163) [inline components/style/themes/default.less]: @card-head-background

File: components/card/style/index.less (hunk 2 above)
Comment (kossel) [inline components/card/style/index.less]: ops you are right, I will fix that

**Comment by kossel** [comment]
Fixed

**Comment by afc163** [comment]
close #5354

**Pull request merged**